52/BlaiseCosico/pomodoro.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timedelta
+import time as time
+
+def header():
+	print(f'\nWelcome to your very own Pomodoro Timer!\n\n'
+		f'--------------INSTRUCTIONS----------------\n'
+		f'Start the Pomodoro timer\n'
+		f'Work until the Pomodoro rings\n'
+		f'Take a short break (5 minutes)\n'
+		f'Keep on working, Pomodoro after Pomodoro, until the task at hand is finished.\n'
+		f'Every 3 Pomodoros take a longer break, (15–30 minutes).\n'
+		f'Enter how many sets do you want. (Each set has 3 Pomodoros)')
+	set_loop(int(input()))
+
+def set_loop(ans):
+	while ans:
```

Comment:
`while` is technically wrong here because you know exactly how often to loop. As ans is an integer and gives the total amount of sets, you just would do a second loop, wouldn't you? No need for a while here and no need for `ans -= 1` in line 24